### PR TITLE
Spelling & Grammar Changes to index.html

### PR DESCRIPTION
**Pull request by CrythicYT**
Some Spelling and Phrasing Changes to the FAQ area.

### CODE REVIEW

**Comment by StoneMcYT** [comment]
Related to :
https://github.com/ASEAN-Build-The-Earth/AseanBTE-Website/issues/95

**Comment by MatthewIsHereYT** [comment]
Will update more tomorrow (18th Nov 2021)

**Comment by tintinkung** [comment]
> Related to : #95

uh ya know ya can like an issue

**Comment by StoneMcYT** [comment]
> 
> 
> Will update more tomorrow (18th Nov 2021)

kk will review when your done btw.

**Comment by Kira272921** [comment]
image
 
 😕 Never heard this term in my entire life

**Comment by XboxBedrock** [comment]
It’s a real term that refers to both north and South America

**Comment by Kira272921** [comment]
> It’s a real term that refers to both north and South America

Ah.. ok

**Comment by Kira272921** [comment]
> Will update more tomorrow (18th Nov 2021)

@MatthewIsHereYT You still working ?

**Comment by Kira272921** [comment]
@CrythicYT, Could you upload your avatar into [this](https://github.com/ASEAN-Build-The-Earth/AseanBTE-Website/tree/main/Assets/Images/avatars) directory ?

File: FAQ/index.html [hunk 1]
```diff
@@ -266,22 +266,33 @@ <h2>FAQ</h2>
...
+                                    <h5 class="faq-title" data-toggle="collapse" data-target="#faqCollapse-9"
+                                        data-aria-expanded="false" data-aria-controls="faqCollapse-9">
+                                        <span class="badge">9</span> 
+                                    </h5>
+                                </div>
+                            </div>
+                            <div id="faqCollapse-9" class="collapse" aria-labelledby="faqHeading-9"
+                                data-parent="#accordion">
+                                <div class="card-body">
+                                    <p>You can watch <a class="link"href="https://www.youtube.com/watch?v=T174gWwD1MU">This Tutorial</a> on how to install the modpack. </p>
```
Comment (Kira272921) [inline FAQ/index.html:292]: 😅 Make `T` into lower case

File: FAQ/index.html [hunk 2]
```diff
@@ -266,22 +266,33 @@ <h2>FAQ</h2>
...
+                                        <span class="badge">9</span> How do I install the BTE Modpack?
                                     </h5>
                                 </div>
                             </div>
                             <div id="faqCollapse-9" class="collapse" aria-labelledby="faqHeading-9"
                                 data-parent="#accordion">
                                 <div class="card-body">
-                                    <p>You can watch on how to install the modpack via this <a class="link"
-                                            href="https://www.youtube.com/watch?v=T174gWwD1MU">link.</a></p>
+                                    <p>You can watch <a class="link"href="https://www.youtube.com/watch?v=T174gWwD1MU">This Tutorial</a> on how to install the modpack. </p>
```
Comment (Kira272921) [inline FAQ/index.html:276]: 😅 Make `T` into lower case

**Review by Kira272921 — CHANGES_REQUESTED** [review]
@CrythicYT, Here are few typography issue, rest of your work is great 👍

**Comment by StoneMcYT** [comment]
This still active?

**Comment by tintinkung** [comment]
I could fix and merge this

**Review by tintinkung — APPROVED** [review]
### good enough 👍

**Pull request merged**